packages/components/Button/src/FAB/__snapshots__/FAB.test.tsx.snap

```diff
@@ -1,5 +1,68 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Custom FAB with no shadow(iOS) 1`] = `
+<View
+  accessibilityLabel="Custom FAB with no shadow(iOS)"
+  accessibilityRole="button"
+  accessibilityState={
+    Object {
+      "disabled": false,
+    }
+  }
+  accessible={true}
+  disabled={false}
+  enableFocusRing={true}
+  focusable={true}
+  iconPosition="before"
+  onBlur={[Function]}
+  onClick={[Function]}
+  onFocus={[Function]}
+  onPress={[Function]}
+  onResponderGrant={[Function]}
+  onResponderMove={[Function]}
+  onResponderRelease={[Function]}
+  onResponderTerminate={[Function]}
+  onResponderTerminationRequest={[Function]}
+  onStartShouldSetResponder={[Function]}
+  style={
+    Object {
+      "alignItems": "center",
+      "alignSelf": "flex-start",
+      "backgroundColor": "#0078d4",
+      "borderColor": "#005a9e",
+      "borderRadius": 9999,
+      "display": "flex",
+      "flexDirection": "row",
+      "justifyContent": "center",
+      "minHeight": 56,
+      "minWidth": 56,
+      "padding": 16,
+      "width": undefined,
```

Comment:
Hmm just noticed that this got added & not sure why. Looking into it

Reply:
Are you talking about this particular width style or the test snapshot itself? The test snapshot addition is expected since there's a new test in the file?

Reply:
Specifically just the "width": undefined line, it wasn't in the original snapshot.

Reply:
DIdn't figure out why this was added but did see that this is in a lot of other snapshot tests, think it should be okay to merge